src/apiData/apiCalls.js

```diff
@@ -1,17 +1,8 @@
+const baseURL = 'https://pokeapi.co/api/v2/pokemon'
 
-const baseURL = 'https://pokeapi.co/api/v2/'
-const getAll = 'https://pokeapi.co/api/v2/pokemon?limit=151'
-const singlePokemon = 'https://pokeapi.co/api/v2/pokemon/'
-const soloPic = 'https://pokeres.bastionbot.org/images/pokemon/1.png'
-
-const apiCalls = async () => {
-  const response = await fetch(`https://pokeapi.co/api/v2/pokemon?limit=151`)
-  return await response.json()
-}
-
-const getIndividualPokemon = async (id) => {
-  const response = await fetch(`https://pokeapi.co/api/v2/pokemon/${id}`)
+const fetchPokemonData = async (query) => {
+  const response = await fetch(`${baseURL}${query}`)
```

Comment:
why are passing arguments in when we only use it once?

Reply:
cuz we are using this function twice to get two diff api data